src/audio/igo_nr/igo_nr.c

```diff
@@ -215,10 +216,9 @@ static inline int32_t set_capture_func(struct comp_dev *dev)
 }
 
 static struct comp_dev *igo_nr_new(const struct comp_driver *drv,
-				   struct sof_ipc_comp *comp)
+				   struct comp_ipc_config *config, void *spec)
```

Comment:
@fuyuntsuo thanks, this last was on my todo list for main branch (but not's not needed in v1.8)

Reply:
no problem. thanks for prompt approval.